src/main/java/pixelperfect/event/EventScheduler.java

```diff
@@ -54,15 +54,27 @@ private void publish(Event event) {
    * models a poisson process, with the specified intensity as lambda parameter.
    * 
    * @param tpf
-   *          The time per frame in seconds, i.e. how many times per second this draw is performed
-   *          and thus the time interval parameter for the poisson process.
+   *          The time per frame in seconds, i.e. the time interval parameter for the poisson
+   *          process.
    */
   public void update(float tpf) {
+    // The random generator should be more efficient and less biased than the Math.random function.
```

Comment:
I know it's "mierenneukerij," but gramatically it should be "This random generator is more efficient and...".